### PR TITLE
logic-building tasks in javascript

### PR DESCRIPTION
**Pull request by vc-jinal**
the string in alphabetical order,
returns the longest string in the array,
find the sum of all the numbers that are divisible by 3 or 5,
returns a new array with only the unique strings (removing duplicates),
sort the strings based on their length, from shortest to longest,
returns the second smallest number in the array.

### CODE REVIEW

File: logic-building/secondSmallestNumber.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//Given an array of strings, write a function to sort the strings based on their length,
```
Comment (vc-mohit) [inline logic-building/secondSmallestNumber.js:1]: is this the correct question of the given answer?

File: logic-building/stringAlphabeticalOrder.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//Write a function that takes a string as input and returns the characters in the string in alphabetical order.
+function alphabeticalOrder(str) {
+	const string = str.split("");
+	// console.log(string);
```
Comment (vc-mohit) [inline logic-building/stringAlphabeticalOrder.js:4]: it is required?

**Review by vc-mohit — APPROVED** [review]

File: logic-building/secondSmallestNumber.js (hunk 1 above)
Comment (vc-jinal) [reply logic-building/secondSmallestNumber.js]: changed question

File: logic-building/stringAlphabeticalOrder.js (hunk 2 above)
Comment (vc-jinal) [reply logic-building/stringAlphabeticalOrder.js]: removed console

**Pull request merged**